src/wallet/bdb.h

```diff
@@ -93,53 +93,66 @@ std::shared_ptr<BerkeleyEnvironment> GetWalletEnv(const fs::path& wallet_path, s
 /** Return wheter a BDB wallet database is currently loaded. */
```

Comment:
not your line but, s/wheter/whether/

Reply:
I think that'd be better to fix in one of the prerequisite PRs or a followup to this whole PR stack.